zaza/openstack/charm_tests/ceph/iscsi/tests.py

```diff
@@ -250,9 +254,11 @@ def run_client_checks(self, test_ctxt):
         """
         self.create_iscsi_target(test_ctxt)
         self.login_iscsi_target(test_ctxt)
+        self.refresh_partitions(test_ctxt)
         self.check_client_device(test_ctxt, init_client=True)
         self.logout_iscsi_targets(test_ctxt)
         self.login_iscsi_target(test_ctxt)
+        self.refresh_partitions(test_ctxt)
         self.check_client_device(test_ctxt, init_client=False)
```

Comment:
Does this need another call to refresh_partitions?

Reply:
Yes, it's needed after every login call.

Reply:
Ok I noticed in later branches there's a refresh_partitions call after the final check_client_device. I wasn't sure if that was needed here.